<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransWindow</class>
 <widget class="QDialog" name="FileTransWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileTransWindow</string>
  </property>
  <widget class="QLabel" name="label_filePath">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_choseFile">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sendFile">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="Bar_progressFile">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_fileState">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>正在发送文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
